lang/contexts.json

```diff
@@ -1,6 +1,7 @@
 {
 	"Paragraph": "Drop-down option label for the paragraph format.",
-	"Heading": "Tooltip and default label for the heading drop-down.",
+	"Heading": "Tooltip for the heading drop-down.",
```

Comment:
Besides, this is not in the scope of this issue. If you want something removed, please create an issue and let people discuss.